test-data/unit/semanal-errors.test

```diff
@@ -980,7 +980,7 @@ e = TypeVar('e', int, str, x=1)   # E: Unexpected argument to TypeVar(): x
 f = TypeVar('f', (int, str), int) # E: Type expected
 g = TypeVar('g', int)             # E: TypeVar cannot have only a single constraint
 h = TypeVar('h', x=(int, str))    # E: Unexpected argument to TypeVar(): x
-i = TypeVar('i', bound=1)         # E: TypeVar 'bound' must be a type
+i = TypeVar('i', bound=1)         # E: Invalid type: try using Literal[1] instead?
```

Comment:
It looks like there is a common trend: if there is an invalid type, it is hard to give a more detailed error "upstream". A potential solution for this may be to introduce a new `TypeOfAny.invalid_type`, so that caller can still choose to give an error for this `Any`. Could you please open an issue for this? (Maybe also link https://github.com/python/mypy/issues/4030 there.)